ui/Config.ui

```diff
@@ -1103,20 +1103,47 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tab_gui">
+      <attribute name="title">
+       <string>GUI behaviour</string>
+      </attribute>
+      <layout class="QGridLayout" name="gridLayout_13">
+       <item row="0" column="0">
+        <widget class="QCheckBox" name="reverseDragBehaviorCheck">
+         <property name="text">
+          <string>Use Left mouse button to drag and change center frequency</string>
```

Comment:
Minor: put "Left" in lower case, and prefix it with & ("Use &left mouse button...") so you can toggle this option using a keyboard accelerator (Alt + l or equivalent).